### PR TITLE
A basic, early implementation of the spooky dimension.

### PR DESCRIPTION
**Pull request by Shrimpstress**
Adds World Traveler as a dependency.

This currently adds a basic version of the spooky dimension with proper lighting values, custom blocks, and a basic biome.

While this is considered incomplete, I would like to get a basic version of it PRed in quickly so that other people can also build onto this dimension if they so desire. I **do** have plans to continue working on this myself, but am welcome to any additions people would like to throw in as long as they make sense.

What this contains:
- A basic implementation of a biome which uses custom grass, dirt, gravel, and sand blocks. It is planned in the future to use a custom stone block as well.
- A basic implementation of a dimension which is made purely of that biome. More biomes will get added over time. The dimension also has light level brightness cut in half, so things are overall darker inside.

### CODE REVIEW

**Review by liach — APPROVED** [review]

File: src/main/java/com/fabriccommunity/spookytime/client/ClientInitializer.java
Comment (Shnupbups) [inline src/main/java/com/fabriccommunity/spookytime/client/ClientInitializer.java]: If you could merge this class with our client initializer we already have, that'd be great!

**Review by Shnupbups — CHANGES_REQUESTED** [review]

File: src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java
Comment (Shnupbups) [inline src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java]: Not sure whether using a star import on the block package is a good idea.

**Review by Shnupbups — CHANGES_REQUESTED** [review]

File: src/main/java/com/fabriccommunity/spookytime/client/ClientInitializer.java
Comment (Shrimpstress) [reply src/main/java/com/fabriccommunity/spookytime/client/ClientInitializer.java]: I’ll do that as soon as possible.

File: src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java
Comment (Shrimpstress) [reply src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java]: This actually wasn’t intentional, IDEA seems to have done it itself. I’ll change the imports when I can.

**Review by Draylar — APPROVED** [review]
Looks great! Going to wait for one or two other approvers before merging, as it's a bigger PR.

**Review by valoeghese — APPROVED** [review]
Merge dis already pls

**Comment by Draylar** [comment]
no

**Review by eyezahhhh — APPROVED** [review]

**Pull request merged**
